### PR TITLE
[Feature][Zeta] Checkpoint support hdfs ha mode

### PR DESCRIPTION
**Pull request by Kyofin**
## Purpose of this pull request

close issue #4676 
## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [  ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Review by davidzollo — DISMISSED** [review]
Thanks for your first contribution. 
You just need to update the code styles, please Run 'mvn spotless:apply' to fix these violations.

**Review by davidzollo — DISMISSED** [review]
Run 'mvn spotless:apply' to fix these violations.

File: seatunnel-engine/seatunnel-engine-storage/checkpoint-storage-plugins/checkpoint-storage-hdfs/src/main/java/org/apache/seatunnel/engine/checkpoint/storage/hdfs/common/HdfsConfiguration.java [hunk 1]
```diff
@@ -69,7 +71,16 @@ public Configuration buildConfiguration(Map<String, String> config)
...
             }
         }
-        // todo support other hdfs optional config keys
+        //  support other hdfs optional config keys
+        config.entrySet().stream()
+                .filter(entry -> entry.getKey().startsWith(SEATUNNEL_HADOOP_PREFIX))
+                .forEach(
+                        entry -> {
+                            String key = entry.getKey().replace(SEATUNNEL_HADOOP_PREFIX, "");
+                            ;
```
Comment (Hisoka-X) [inline seatunnel-engine/seatunnel-engine-storage/checkpoint-storage-plugins/checkpoint-storage-hdfs/src/main/java/org/apache/seatunnel/engine/checkpoint/storage/hdfs/common/HdfsConfiguration.java]:
```suggestion
```

**Comment by hailin0** [comment]
please  update docs

**Comment by EricJoy2048** [comment]
Please rerun `mvn spotless:apply` or ` sh tools/spotless_check/pre-commit.sh` . Thanks.

**Comment by ic4y** [comment]
Waiting for CI/CD.

**Comment by Kyofin** [comment]
> Waiting for CI/CD.

need  CI/CD.

**Comment by ic4y** [comment]
image
@Kyofin  Please use `mvn spotless:apply` to format the code.

**Comment by ic4y** [comment]
@wu-a-ge 
image

**Review by ic4y — APPROVED** [review]

**Review by liugddx — APPROVED** [review]
+1

**Comment by liugddx** [comment]
@davidzollo PTAL

**Pull request merged**